www/content/articles/analyse_et_optimisation/index.md

```diff
@@ -0,0 +1,810 @@
+
+Mais, de nos jours, les claviers informatiques n’ayant pas de contrainte de ce
+type, cette métrique du SHU n’est plus utilisée pour mesurer la qualité d’une
+disposition — voire, elle est souvent utilisée dans l’autre sens : un SHU élevé
+caractérise une disposition « rolly », c’est-à-dire privilégiant les roulements.
+C’est le cas de la très grande majorité des dispositions modernes, dont Workman
+et Ergo‑L.
+
+> In designing Workman, I preferred a high SHU (low alternation) over a low SHU
+> (high alternation). I think high alternation is beneficial if you’re typing on
+> mechanical typewriters but not necessarily on modern keyboards. […]
+
+> It is much more efficient to ride the momentum of a single arm or wrist stroke
+> and type a combo rather than just one key.
+
+— [Kevin DeLoach][kdeloach], disposition [Workman][]
+
+C’est un constat qui est également partagé par bon nombre de Bépoètes.
+
+> La frappe d’un roulement facile peut être plus rapide, plus confortable et
+> moins sujette à la faute de frappe qu’une alternance de mains.
+
+— extrait des notes de conception de [Béop][]
+
+Et c’est un sujet qui a été longuement détaillé par [Nuclear-Squid][] dans sa
+conférence « [Ergonomie clavier : l’ère post-Dvorak][cdl2023-slides] »
+([vidéo][cdl2023-video]).
+
+Attention : les dispositions de clavier qui privilégient les roulements prennent
+le risque d’avoir un taux élevé de mauvaises redirections.
+ 
+### Distance parcourue ?
+
+Une métrique originale consiste à mesurer la distance parcourue par les doigts
+pendant la saisie d’un texte. Elle a été rendue populaire par CarpalX et KLA :
+facile à calculer, et c’est toujours amusant de savoir combien de kilomètres nos
+doigts parcourent pour saisir un texte. :-)
+
+La communauté Ergonaute n’utilise pas cette métrique car elle n’est pertinente
+que si l’on suppose que tous les déplacements de doigt se valent — ce qui est
+évidemment faux (voir le paragraphe sur la [heatmap](#la-heatmap)).
+
+### Note globale ?
+
+Certains analyseurs de layout proposent une note globale, qui est une moyenne
+pondérée de plusieurs critères plus objectifs. Si vous connaissez les critères
+utilisés et êtes d’accord avec leur pondération, cette note globale peut donner
+une évaluation générale appréciable lors des premières phases d’optimisation.
+
+Mais il y a deux choses importantes à garder à l’esprit :
+
+- une note globale ne rend pas compte des pires défauts de la disposition : ils
+  sont soit noyés dans la moyenne, soit complètement ignorés ;
+- quelle que soit sa qualité, une note globale ne sera jamais une bonne cible
+  d’optimisation.
+
+> Lorsqu'une métrique devient un objectif, elle cesse d'être une bonne métrique.
+
+— [loi de Goodhart](https://fr.wikipedia.org/wiki/Loi_de_Goodhart)
+
+
+Optimiser pour une note globale ?
+--------------------------------------------------------------------------------
+
+Ça peut être contre-intuitif, mais optimiser pour une note globale est voué à
```

Comment:
```suggestion
Cela peut être contre-intuitif, mais optimiser pour une note globale est voué à
```

Reply:
C'est comme ça que je comprends ta pratique sur le site. Pour cet usage, on navigue souvent entre deux écueils : « ça » peut paraître trop oral à l'écrit et « ce/cela », trop formels pour le contexte, même écrit.

Reply:
S’il n’y a pas de règle précise pour ça, je préfère conserver la forme actuelle. Et s’il faut faire preuve de cohérence, pour les articles signés de mon pseudo je préfèrerai toujours la forme moins soutenue.

Reply:
« Ça » est inélégant, et le reste de l’article n’a pas un ton suffisamment informel pour justifier le langage oral à mon sens, mais c’est une question de style donc tu fais bien comme tu veux :)

Reply:
Dans le texte actuel il y a 4 `cela` et 24 `ça`. Je m’étonnais juste de la remarque pour ce paragraphe-là en particulier.

S’il y a une règle, je veux bien l’apprendre. Si c’est juste une question d’inélégance, je m’en accommode. :-)

Reply:
J’ai suggéré la modif sur une seule occurrence pour tâter le terrain, et bien m’en a pris, semble-t-il ;-)

J’ai beau chercher, je ne trouve pas de règle, si ce n’est que les dictionnaires marquent le terme comme familier — ce dont tu te fiches. Je pense que tu peux fermer le fil, donc (il semble que je ne le puisse pas).